static/website/templates/get-question.html

```diff
@@ -149,10 +148,26 @@ <h4><u>Answers:</u></h4>
             </div>
         </div>
 
-        <div class="answer {% if user|is_moderator %}editable{% endif %}" id="answer{{answer.id}}" style="margin-left: 63px;">                               
-            
+        <div class="answer {% if MODERATOR_ACTIVATED or user|can_edit:answer %}editable{% endif %}" id="answer{{answer.id}}" style="margin-left: 63px;">                               
             <div class="body" id="body{{ answer.id }}" style="padding-bottom: 12px; padding-top: 15px">
-                <p>{{ answer.body| safe }}</p>
+                {% if MODERATOR_ACTIVATED or user|can_edit:answer and answer|havenot_comments %}
```

Comment:
The logic here is:

```
if MODERATOR_ACTIVATED:
   show the form
else if user|can_edit:answer and answer|havenot_comments:
   show the form
```

so havenot_comments is checked only for the second part. is this logic correct?

Reply:
Yes, this is correct.